psalm-baseline.xml

```diff
@@ -2192,10 +2191,6 @@
     </PropertyNotSetInConstructor>
   </file>
   <file src="lib/Doctrine/ORM/Query/AST/Functions/SubstringFunction.php">
-    <InvalidScalarArgument occurrences="2">
-      <code>$optionalSecondSimpleArithmeticExpression</code>
-      <code>$sqlWalker-&gt;walkSimpleArithmeticExpression($this-&gt;firstSimpleArithmeticExpression)</code>
-    </InvalidScalarArgument>
```

Comment:
Some of these would need to be "moved" to a baseline specific to 2.13

Reply:
See https://github.com/doctrine/orm/pull/9343

Reply:
I could also move them to psalm.xml.dist and add a comment that this applies to DBAL 2 only. That should stabilize the baseline as well.